langserver/internal/refs/testdata/http-request-headers.go

```diff
@@ -0,0 +1,12 @@
+package main
+
```

Comment:
I actually had these as not ending in `.go` but rather `.test` so that lint/etc tools would not run on them, but ironically that made some JS linter try to run them :) I will leave it as-is for now, since it's minor.